docs/book/src/component-config-tutorial/testdata/project/config/manager/controller_manager_config.yaml

```diff
@@ -1,5 +1,5 @@
-apiVersion: controller-runtime.sigs.k8s.io/v1alpha1
-kind: ControllerManagerConfig
+apiVersion: config.tutorial.kubebuilder.io/v2
+kind: ProjectConfig
 health:
   healthProbeBindAddress: :8081
 metrics:
```

Comment:
@tomasfabian I am re-open this one. 
After re-check I see that you are right here.

The example code leads to a misunderstanding if, you create a CustomType you need to update the file to use it instead. Could you please just assign the CLA so we get this one merged? 

/lgtm
/approved
/ok-to-test

Reply:
Hi @camilamacedo86, thank you very much for accepting this PR. 
I tried to assign to CLA. Is it ok now?